content/docs/Configuration/examples.md

```diff
@@ -229,6 +229,27 @@ jobs:
   ```
 {{< /details >}}
 
+{{< details title="Tag cloud resources in Testing Farm" >}}
+Tag cloud resources in Testing Farm to a specific team or project.
+Make sure to update `sst_change_me` to your RHEL SST name or name
+of the project. If not set, cloud costs are reported against
+`Packit Service`. The `BusinessUnit` key name is required, please
+do not change it.
+```yaml
+- job: tests
+  trigger: pull_request
+  targets:
+    - fedora-all
+  # Tag cloud resources for tmt
+  tf_extra_params:
+    environments:
+      - settings:
+          provisioning:
```

Comment:
Are we sure that YAML won't screw up this indentation?

Reply:
how screw up?

We tested it here, it cannot be on the same level as settings as it would be another key, we need it nested:

```
❯ cat test.yaml
enviornments:
  - settings:
      provisioning:
        tags:
          BusinessUnit: sst

❯ yq < test.yaml
{
  "enviornments": [
    {
      "settings": {
        "provisioning": {
          "tags": {
            "BusinessUnit": "sst"
          }
        }
      }
    }
  ]
}

```

Reply:
Oh yeah, sorry, I just got a bit confused and thought it's supposed to be in the fmf syntax :smile:

Reply:
np, so resolving :)